src/pages/MetricDetail.svelte

```diff
@@ -2,10 +2,116 @@
   import { getMetricData } from "../state/api";
 
   export let params;
+
   const metricDataPromise = getMetricData(params.app, params.metric);
 </script>
 
 {#await metricDataPromise then metric}
   <h1>{metric.name}</h1>
-  <p>{metric.description}</p>
+  <h2>{metric.description}</h2>
```

Comment:
This should not be a header, the previous paragraph was correct.

```suggestion
  <p>{metric.description}</p>
```